"""
다음 코드는 3을 0으로 나누고 있습니다. 나눗셈 연산은 0으로 나눌 수 없기 때문에 문제의 코드를 실행하면 에러가 발생합니다. 에러의 이름을 확인하고 try except문으로 코드의 2번째 줄을 감싸서 에러가 발생할 경우 4번째 줄이 출력되도록 만들어 보세요.

a = 3/0

print("0으로 나눌 수 없습니다.")
"""

try:
	a = 3/0
except ZeroDivisionError: #나눗셈 연산에서 분모가 0일때의 에러
	print("0으로 나눌 수 없습니다.")